src/components/HintPage.jsx

```diff
@@ -1,19 +1,49 @@
-import React from 'react'
+import React, { useState } from 'react'
 import Button from 'cozy-ui/transpiled/react/Button'
 import { translate } from 'cozy-ui/transpiled/react/I18n'
 import Input from 'cozy-ui/transpiled/react/Input'
+import compose from 'lodash/flowRight'
+import { withClient } from 'cozy-client'
+import Alerter from 'cozy-ui/transpiled/react/Alerter'
 
 const DumbHintPage = props => {
-  const { t } = props
+  const { t, client, history } = props
+  const [hint, setHint] = useState('')
+  const [loading, setLoading] = useState(false)
```

Comment:
s/loading/saving/ would be clearer I think, at first I thought you were loading the hint, not saving it.

Reply:
See https://github.com/cozy/cozy-passwords/pull/1/commits/ec2653828c8c69786d8b3294c63ad8794729fa1f